### PR TITLE
Tower action version updating 2

### PR DESCRIPTION
**Pull request by gq1**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/variantcalling/tree/main/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by gq1** [comment]
@muffato
I merged your PR but the Tower submission failed without any error. Probably the version number is wrong. 
https://github.com/sanger-tol/variantcalling/actions/runs/5865831388/job/15903577653 

Because I can't see any log files for the action and that is why I keep the log files as well.

Because the workflow file name changed, they will be listed as a different workflow although the workflow name didn't change. https://github.com/sanger-tol/variantcalling/actions

We can't manually trigger the new workflow yet because they are not in the main branch.

Can you have a quick look and then I can merge to the `dev` to see it works or not?

**Comment by muffato** [comment]
The version bump feels strange. I usually only see `@v1`, `@v2` etc in yml files, and I thought it meant GitHub would automatically pick up the latest minor version.
I think the problem is that their repo has a `v2` tag. My feeling is that it should only have `v2.0.0` so that GitHub could select the version. Let me ask on their Slack

File: .github/workflows/sanger_test.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Launch workflow via tower
-        uses: seqeralabs/action-tower-launch@v2
+        uses: seqeralabs/action-tower-launch@2.0.0
```
Comment (muffato) [inline .github/workflows/sanger_test.yml:12]: Can we first try without this version change ? I only ever see the `@v2` syntax, not the`@2.0.0` one

Comment (gq1) [reply .github/workflows/sanger_test.yml:12]: https://github.com/marketplace/actions/action-tower-launch
The green button on the top right said using `@2.0.0`. I can't see why it doesn't work use v2, that is why I just give a try.

**Comment by gq1** [comment]
https://github.com/sanger-tol/variantcalling/actions/runs/5866300535/job/15904836219
no any error message here and no logs files generated as well.

**Comment by muffato** [comment]
Can you compare the yml files to https://github.com/nf-core/funcscan/blob/1.1.3/.github/workflows/awstest.yml and https://github.com/nf-core/funcscan/blob/1.1.3/.github/workflows/awsfulltest.yml, which was just released a few days ago ?

**Comment by muffato** [comment]
or https://github.com/nf-core/funcscan/blob/1.1.3/.github/workflows/awstest.yml and https://github.com/nf-core/funcscan/blob/1.1.3/.github/workflows/awsfulltest.yml

**Comment by gq1** [comment]
> Can you compare the yml files to https://github.com/nf-core/funcscan/blob/1.1.3/.github/workflows/awstest.yml and https://github.com/nf-core/funcscan/blob/1.1.3/.github/workflows/awsfulltest.yml, which was just released a few days ago ?

Here is their finished job. We just don't have any error message.
https://github.com/nf-core/funcscan/actions/runs/5832636289/job/15818453876

**Comment by gq1** [comment]
@muffato I am going to merge this PR now and hope everything will work this time. Then we can apply to other repos as well.

Just need to delete the pipeline variable, which is not the full repo url.

We may add the upload artifacts step as well.

**Pull request merged**